compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -1078,8 +1078,12 @@ declare_lint! {
     /// [undefined behavior]: https://doc.rust-lang.org/reference/behavior-considered-undefined.html
     /// [issue #27060]: https://github.com/rust-lang/rust/issues/27060
     pub UNALIGNED_REFERENCES,
-    Allow,
+    Warn,
```

Comment:
Can you also update the documentation a few lines above to remove the paragraph about it being `allow`?  And maybe switch the link to the new tracking issue (#82523)?

Reply:
Good point, done.